src/components/Content/Content.component.jsx

```diff
@@ -12,14 +12,13 @@ const useStyles = makeStyles({
 });
 
 export default function Content() {
+  const globalContext = useGlobal();
   const [videos, setVideos] = useState([]);
-  const searchContext = useContext(SearchContext);
   const classes = useStyles();
 
   useEffect(() => {
-    console.log(searchContext.searchText);
-    getVideos(setVideos, searchContext.searchText);
-  }, [searchContext.searchText]);
+    getVideos(setVideos, globalContext.searchState);
```

Comment:
Instead of passing a function to `getVideos`, return the result so you can use the result where you actually want it. It'll keep your function agnostic/reusable whenever it is used. As I mentioned on the previous review, this looks already like a custom hook. I suggest you to converting it to one so you can use it as:

```javascript
const { searchState } = globalContext;
const videos = useYoutubeVideos(searchState);
```

You'll be removing the need of the useState([]) entirely with this approach.